language/English/strings.po

```diff
@@ -8342,7 +8346,11 @@ msgctxt "#19178"
 msgid "Show channel information when switching channels"
 msgstr ""
 
-#empty string with id 19179
+#. Used as extension header on recordings window
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19179"
+msgid "Deleted recordings trash"
```

Comment:
Just been checking the usage of English for the strings.po additions and wasn't sure about this one, could you confirm where & how it's used? a screenshot would be great if you have the time.